shell/packages/sandstorm-ui-powerbox/powerbox.html

```diff
@@ -0,0 +1,80 @@
+<template name="powerboxRequest">
+  <h4>Powerbox request</h4>
+  {{#if showWebkeyInput}}
+    <form id="powerbox-request-form">
+      <label>Please input an api token: </label><input name="token" type="text" id="powerbox-request-input"><br>
+      <button class="submit">Import token</button>
+    </form>
+    {{#if webkeyError}}
+    <div class="error">{{webkeyError}}</div>
+    {{/if}}
+  {{else}}
+    <h4>Connect this grain with:</h4>
```

Comment:
Yeah, I agree the copy here is tricky.  "Attaching" might make sense in the context of Rocket.Chat, but is probably less suitable for e.g. a folder of grains.  "Connect" is reasonably good for things like static publishing, or remote hosts to connect to (e.g. RSS feeds) but not super great in general.  I do like that it hints at the persistent nature of these connections, though, which isn't necessarily obvious from the get-go.

"Choose a grain:" is better for this specific case, but "grain" might be confusing when frontendrefs enter the mix, like if the app is requesting an IpNetwork or IpInterface, which may be implemented by either the frontend or by apps.  We could have a small dictionary for the various Sandstorm-recognized types, but then non-Sandstorm-aware interfaces wind up disadvantaged, or we have .  Maybe the request should specify an overriding noun phrase, but then we have to change the request/offer interfaces?  Maybe something else?

The text I originally wrote tries to dodge the "what noun should we use?" question entirely.  I want to have some sort of verb here indicating what the user should do next, and ideally the noun would be one of the items in the list.  "Grain" is almost right, except when it's not.

Maybe this could just say "Select one:" and rely on the context provided by the app making the request?  I think I like this option the best, though it's also the least specific/helpful.

@neynah if you have any ideas here, I'm all ears